src/hotspot/share/jfr/jni/jfrJniMethodRegistration.cpp

```diff
@@ -44,6 +44,7 @@ JfrJniMethodRegistration::JfrJniMethodRegistration(JNIEnv* env) {
       (char*)"destroyJFR", (char*)"()Z", (void*)jfr_destroy_jfr,
       (char*)"emitEvent", (char*)"(JJJ)Z", (void*)jfr_emit_event,
       (char*)"getAllEventClasses", (char*)"()Ljava/util/List;", (void*)jfr_get_all_event_classes,
+      (char*)"getClassId", (char*)"(Ljava/lang/Class;)J", (void*)jfr_class_id,
       (char*)"getClassIdNonIntrinsic", (char*)"(Ljava/lang/Class;)J", (void*)jfr_class_id,
```

Comment:
Please remove the getClassIdNonIntrinsic entry, thanks. In addition, can you also remove the now abandoned entry point on the Java side, jdk.jfr.internal.JVM.getClassIdNonIntrinsic. And the entry point in jfr/jni/jniMethod.hpp | .cpp. Thanks.

Reply:
removed.